.changeset/shiny-chefs-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hopper-ui/components": patch
+---
```

Comment:
<img width="1444" alt="image" src="https://github.com/user-attachments/assets/b3bd5c30-1410-4082-b531-032b6f4e017c">
There's some weird issue about some things not being focusable

Reply:
seems to be some weird case that wouldn't happen elsewhere. Even if it does, it's something done internally in RAC.